maligned.go

```diff
@@ -29,6 +31,7 @@ func main() {
 	}
 
 	var conf loader.Config
+	conf.TypeChecker.Sizes = types.SizesFor("gc", env("GOARCH", runtime.GOARCH))
```

Comment:
You can just use types.SizesFor(build.Default.Compiler, build.Default.GOARCH).

Reply:
Nice. Done